src/test/ssmDocument/commands/openDocumentItem.test.ts

```diff
@@ -0,0 +1,86 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SSM } from 'aws-sdk'
+
+import * as assert from 'assert'
+import * as sinon from 'sinon'
+import * as vscode from 'vscode'
+
+import { openDocumentItem } from '../../../ssmDocument/commands/openDocumentItem'
+import { DocumentItemNode } from '../../../ssmDocument/explorer/documentItemNode'
+
+import * as picker from '../../../shared/ui/picker'
+import { MockSsmDocumentClient } from '../../shared/clients/mockClients'
+import { FakeAwsContext } from '../../utilities/fakeAwsContext'
+
+describe('openDocumentItem', async () => {
+    let sandbox: sinon.SinonSandbox
+    beforeEach(() => {
+        sandbox = sinon.createSandbox()
+    })
+
+    afterEach(() => {
+        sandbox.restore()
+    })
+
+    const rawContent: SSM.Types.GetDocumentResult = {
+        DocumentFormat: 'json',
+        DocumentType: 'Command',
+        Name: 'testDocument',
+        Content: `{
+            "schemaVersion": "2.2",
+            "mainSteps": [
+
+            ]
+        }`,
+    }
+
+    const fakeDoc: SSM.Types.DocumentIdentifier = {
+        Name: 'testDocument',
+        DocumentFormat: 'json',
+        DocumentType: 'Command',
+        Owner: 'Amazon',
+    }
+
+    const fakeAwsContext = new FakeAwsContext()
+
+    const fakeFormatSelection = [
+        {
+            label: 'JSON',
+            description: 'Open document with format JSON',
```

Comment:
Open document with format JSON => Open document in JSON format?

Reply:
Will make this change